content/blog/2024/csswg-july.md

```diff
@@ -0,0 +1,448 @@
+---
+title: CSS Working Group updates for June & July
+sub: What I've been working on as an Invited Expert
+date: 2024-07-23
+image:
+  src: blog/2024/csswg-june.jpg
+  alt: >
+    A stepped gradient of a pink hue
+    in 2% lightness increments
+    from 100% to 58%,
+    labeled 'spec'
+tags:
+  - CSS
+  - CSSWG
+  - Cascade Layers
+  - CSS Scope
+  - CSS Nesting
+  - CSS Mixins & Functions
+  - Color
+  - Container Queries
+  - Open Source
+summary: >
+  The CSS Working Group has regular face-to-face meetings
+  (hybrid online/in-person) throughout the year,
+  and they always result in a flurry of activity!
+  Here's a rundown of some highlights
+  from the last few months,
+  with a focus on the features I maintain.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+{% callout %}
+This is hopefully the first post in a series,
+providing updates around my work
+on the CSS Working Group.
+
+These contributions take a lot of time and effort.
+If you're interested in supporting
+our open-source work around CSS,
+consider becoming a [GitHub sponsor](https://github.com/sponsors/oddbird),
+or contributing to our
+[Open Collective](https://opencollective.com/oddbird-open-source).
+{% endcallout %}
+
+## Container Queries & Units
+
+spec:
+~~[Containment, Level 3](https://www.w3.org/TR/css-contain-3/)~~
```

Comment:
on hover, the strikethrough also grows, lol. I didn't think that would happen.

Reply:
whoa, interesting. Seems to be Safari only.